src/intTest/org/onflow/flow/sdk/transaction/TransactionIntegrationTest.kt

```diff
@@ -71,89 +92,143 @@
     fun `Can get block header by id`() {
         val accessAPI = IntegrationTestUtils.newMainnetAccessApi()
 
-        val latestBlock = accessAPI.getLatestBlock(true)
+        val latestBlock = IntegrationTestUtils.handleResult(
+            accessAPI.getLatestBlock(true),
+            "Failed to get latest block"
+        )
+
         assertThat(latestBlock).isNotNull
-        val blockHeaderById = accessAPI.getBlockHeaderById(latestBlock.id)
+
+        val blockHeaderById = IntegrationTestUtils.handleResult(
+            accessAPI.getBlockHeaderById(latestBlock.id),
+            "Failed to get block header by ID"
+        )
+
```

Comment:
Thanks Lea. Appreciate the attention. I don't agree with the bot in case because it's focused on our tests. Maybe it's more case by case, and happy to get more clarity.